### PR TITLE
move citation out of blockquote per spec

### LINKED ISSUE #28401: Example given on blockquote page is incorrect according to the HTML spec
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

### What specific section or headline is this issue about?

Try it

### What information was incorrect, unhelpful, or incomplete?

The example given places the attribution for the quotation inside `blockquote` element, which incorrect according to the HTML specification.

### What did you expect to see?

The example should place the quotation attribution outside the `blockquote` element, eg. in `figcaption` element:

```
<figure>
  <blockquote cite="https://www.huxley.net/bnw/four.html">
    <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
  </blockquote>
  <figcaption>—Aldous Huxley, <cite>Brave New World</cite></figcaption>
</figure>
```

### Do you have any supporting links, references, or citations?

The HTML specification (https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element) states: “Attribution for the quotation, if any, must be placed outside the `blockquote` element.” 

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/blockquote`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/blockquote/index.md
* Last commit: https://github.com/mdn/content/commit/e04d8d2766c468f149445c0bf438d09f9b2d188c
* Document last modified: 2023-06-13T08:10:47.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
fixes https://github.com/mdn/content/issues/28401

### CODE REVIEW

**Comment by Josh-Cena** [comment]
This significantly changes the layout of the quote and I'm not sure if anyone would be doing this. Usually, when rendering a quote, you would expect the attribution to be styled as if it's part of the quote (gray color, left vertical line, right aligned, etc.), and not as if it's any other paragraph. This was why, before https://github.com/mdn/interactive-examples/pull/2562, it was wrapped in another figure, so styles can be applied to the `figure`. `figure`/`figcaption` is also the spec-recommended structure when writing attribution. However based on https://github.com/mdn/content/issues/28401#issuecomment-1666396747, it seems this actually degrades screen reader performance? Maybe we should get the spec authors to clarify?

**Comment by estelle** [comment]
> However based on [mdn/content#28401 (comment)](https://github.com/mdn/content/issues/28401#issuecomment-1666396747), it seems this actually degrades screen reader performance? 

That's why i didn't use figcaption.

> changes the layout of the quote

I updated the CSS and HTML to make it look like a blockquote without adding any semantics for a screenreader to find.

**Comment by Josh-Cena** [comment]
OK, I'm +1 on the current form, but do we want to clarify in content that although the spec says to use `figure`, it's actually bad for some screen readers?

File: live-examples/html-examples/text-content/css/blockquote.css [hunk 1]
```diff
@@ -1,10 +1,7 @@
-blockquote {
-  margin: 0;
-}
-
-blockquote p {
+div:has(> blockquote) {
+  background-color: ##ededed;
```
Comment (Josh-Cena) [inline live-examples/html-examples/text-content/css/blockquote.css]:
```suggestion
  background-color: #ededed;
```

**Comment by estelle** [comment]
> OK, I'm +1 on the current form, but do we want to clarify in content that although the spec says to use `figure`, it's actually bad for some screen readers?

Not in the interactive example, as I assume most people don't actually read the content in the example.

**Comment by Josh-Cena** [comment]
@sideshowbarker Since you approved https://github.com/mdn/content/pull/34813, could you take a quick look at this too? (You are probably also more knowledgeable about HTML than me.)

**Comment by sideshowbarker** [comment]
> _@sideshowbarker Since you approved [mdn/content#34813](https://github.com/mdn/content/pull/34813), could you take a quick look at this too? (You are probably also more knowledgeable about HTML than me.)_
>…
> _Maybe we should get the spec authors to clarify?_

1. Given that at https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element:the-blockquote-element-4 the spec text itself is unambiguous:

   > _Attribution for the quotation, if any, must be placed outside the [blockquote](https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element) element._

2. Over many years of discussions about the spec that I’ve followed (going back 17 years or more), I’ve seen this debated many times — with some people expressing very strong disagreement with it (including some prominent “standardista” influencers) — but the resolution has nevertheless always been to keep that attribution-must-be-placed-outside-the-blockquote requirement in there

…then I’m certain the answer from the spec authors would just be, _“That’s what the spec says”_. I think they’d probably even not suggest that anyone open a new issue if they disagree — because there have already been issues for it, and they’ve been resolved, and it doesn’t seem like there’s any new information which merits re-visiting those.

So I think as far as MDN goes, we have an obligation to conform to the spec requirements, and I’ll go ahead and merge this.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**